### DISCUSSION TITLE
The terminal does not open

### DISCUSSION THREAD

**Discussion by bbslipers** [SpartanJ/ecode#106] [Q&A]
![image](https://github.com/SpartanJ/ecode/assets/20872380/7ed87c4c-a6ef-44d3-87c7-018ee06795c2)

OS Windows Server 2016

**Comment by SpartanJ** [comment]
Unfortunately I can't do much about it. Old Windows versions don't provide the required API to support terminals in ecode ([CreatePseudoConsole](https://learn.microsoft.com/en-us/windows/console/createpseudoconsole)). Terminal support is available in all Windows 10 (RS5) versions and above.
